### PR TITLE
(10964): Update all markdowns for attributes and computed values according to AWS

### PR DESCRIPTION
**Pull request by zhiqiangjin**

### CODE REVIEW

File: website/docs/d/aviatrix_account.html.markdown [hunk 1]
```diff
@@ -12,7 +12,7 @@ Use this data source to get the Aviatrix cloud account for use in other resource
 ## Example Usage
 
 ```hcl
-# Create Aviatrix account data source
+# Create an Aviatrix account data source
```
Comment (arvindsree) [inline website/docs/d/aviatrix_account.html.markdown]: Remove the "create an" . It isn't a resource

File: website/docs/d/aviatrix_caller_identity.html.markdown [hunk 2]
```diff
@@ -12,7 +12,7 @@ Use this data source to get the Aviatrix caller identity for use in other resour
 ## Example Usage
 
 ```hcl
-# Create Aviatrix caller identity data source
+# Create an Aviatrix caller identity data source
```
Comment (arvindsree) [inline website/docs/d/aviatrix_caller_identity.html.markdown]: same

File: website/docs/d/aviatrix_gateway.html.markdown [hunk 3]
```diff
@@ -12,7 +12,7 @@ Use this data source to get the Aviatrix gateway for use in other resources.
 ## Example Usage
 
 ```hcl
-# Create Aviatrix gateway data source
+# Create an Aviatrix gateway data source
```
Comment (arvindsree) [inline website/docs/d/aviatrix_gateway.html.markdown]: same

**Review by arvindsree — APPROVED** [review]

**Review by inc1t3Ful — APPROVED** [review]
Just double checking, are there no other computed values for other resources?

**Comment by arvindsree** [comment]
The parameters that are optional and computed like eip, ha_eip etc that are generated( if not used) should also be added in the attribute reference section

**Pull request merged**